packages/shared/src/components/Feed.tsx

```diff
@@ -123,11 +123,12 @@ export default function Feed<T>({
   const insaneMode = !forceCardMode && listMode;
   const numCards = currentSettings.numCards[spaciness ?? 'eco'];
   const isSquadFeed = feedName === 'squad';
+  const { shouldUseFeedLayoutV1 } = useFeedLayout({ feedName });
```

Comment:
Why do you need to pass it here, the context will have it right?

Reply:
Yup the context should have it, that's why we removed it from the hook props

Reply:
This component is not yet under that context, so it is not familiar with the value.